modules/icinga/manifests/passive_check.pp

```diff
@@ -40,7 +40,7 @@
 define icinga::passive_check (
   $service_description,
   $host_name,
-  $ensure                  = 'present',
+  $ensure                  = 'absent',
```

Comment:
Please revert this to "present" - it is the safer default ; ) (We may have entries which have no "ensure" defined...)

Reply:
Ok, good catch on that. I will make sure that we revert it in case of config files created without the explicit value.